### ISSUE TITLE
Simple adjustment for preventing kick abuse

### ISSUE THREAD

**Issue by minimac1** [arcadia-redux/12v12#382]
The intention of kicking allies, is to be able to kick people that are trolling or feeding but too often we see people getting kicked for doing poorly on a losing team. We could try to do some more complicated detection for whether or not the person getting kicked actually has an impact on the game but a much simpler attempt to fix could be to just raise the number of votes required to kick. 

[Try starting at 9](https://github.com/arcadia-redux/12v12/blob/eefc321140ab999d627ba7985a7e625e39892f5c/game/scripts/vscripts/common/voting_to_kick.lua#L7) without parties and[ when adjusting the required votes](https://github.com/arcadia-redux/12v12/blob/eefc321140ab999d627ba7985a7e625e39892f5c/game/scripts/vscripts/common/voting_to_kick.lua#L152) go to `math.floor(max_voices_in_team*3/4)`. This will mean that the team needs more confidence that the user is actually a troll (which shouldn't be an issue if thats true) and otherwise will avoid less teams simply kicking because they are losing and upset. 

To summarise, this should be about determining confidently that the user is definitely a troll, not simply that a majority vote wants to get rid of them.
